### ISSUE TITLE
Profile Tooltip Modifier doesn't work

### ISSUE THREAD

**Issue by bsolar17** [RaiderIO/raiderio-addon#148]
In the premade group finder hovering on a group shows either your own Raider.IO profile or the group leader profile depending on configuration. This works.

Another configuration allows to switch between the two on the fly by pressing a modifier key. This does not work: the modifier keys have no effect whatsoever.

**Comment by Vladinator89** [comment]
I agree that this is a issue in cases where the hovered person has no profile, there is no tooltip, and the modifier doesn't show your own (if its in that configuration), and it might also do the same if you hover someone (but the config shows your own profile first) and then you hold the modifier and if they have no profile it might not swap or update appropriately.

It's a bit of a niche but definitely a bug that we should look into and make sure the tooltip behaves as it should. The temporary fix is to hold the modifier and move the mouse over the person, as the moving event should trigger the drawing of the tooltip, and if you held down a modifier or not in advance affects it to behave as expected, it's most likely just the swapping part that isn't behaving as expected currently - when the mouse is already hovering the person and you just press the modifier to swap.

**Comment by joeykoko** [comment]
I keep getting this Error 

4x [ADDON_ACTION_BLOCKED] AddOn 'RaiderIO' tried to call the protected function 'RaiderIO_ProfileTooltipAnchor:EnableMouse()'.
[string "@!BugGrabber/BugGrabber.lua"]:485: in function <!BugGrabber/BugGrabber.lua:485>
[string "=[C]"]: in function `EnableMouse'
[string "@RaiderIO/core.lua"]:6456: in function <RaiderIO/core.lua:6455>
[string "@RaiderIO/core.lua"]:6472: in function <RaiderIO/core.lua:6464>
[string "=(tail call)"]: ?
[string "=[C]"]: ?
[string "=[C]"]: ?

Locals:
_ = Frame {
 RegisterEvent = <function> defined @!BugGrabber/BugGrabber.lua:491
 0 = <userdata>
 UnregisterEvent = <function> defined @!BugGrabber/BugGrabber.lua:491
 SetScript = <function> defined @!BugGrabber/BugGrabber.lua:491
}
event = "ADDON_ACTION_BLOCKED"
events = <table> {
 ADDON_ACTION_BLOCKED = <function> defined @!BugGrabber/BugGrabber.lua:557
 ADDON_ACTION_FORBIDDEN = <function> defined @!BugGrabber/BugGrabber.lua:557
 PLAYER_LOGIN = <function> defined @!BugGrabber/BugGrabber.lua:551
 LUA_WARNING = <function> defined @!BugGrabber/BugGrabber.lua:566
 ADDON_LOADED = <function> defined @!BugGrabber/BugGrabber.lua:511
}

Any help would be appreciated .
